queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
```

Comment:
https://github.com/iBel251/vet-clinic/blob/b84bd8840ebe991cb8cd3f5ddd4888d7c41e2432/schema.sql#L2-L11

- [OPTIONAL] Great job so far 👏🏻 however you can make the id column as primary key not null and autoincremented
 eg:
```sql
    id int NOT NULL IDENTITY, PRIMARY KEY (id)
```
- You can visit this link about how to set [auto increment primary key in PostgreSQL?](https://stackoverflow.com/questions/7718585/how-to-set-auto-increment-primary-key-in-postgresql) for more information 🤓